common/src/main/java/com/wynntils/core/config/ConfigManager.java

```diff
@@ -35,7 +38,7 @@ public class ConfigManager {
     private static final List<ConfigHolder> CONFIG_HOLDERS = new ArrayList<>();
     private static File userConfig;
     private static JsonObject configObject;
-    private static Gson gson;
+    static Gson gson;
```

Comment:
Not 100% happy with exposing fields like this, but I'm willing to turn a blind eye to for this time. :)

Reply:
We need this gson instance so it has the same adapters both in ConfigHolder and ConfigManager.

Reply:
If you can suggest a better alternative, let me know.

Reply:
The standard solution is to make a public accessor :-). I know it adds boilerplate but I think it's good to try to stick with the practice of keeping fields private. (As Mojang most definitely not do :))